projects/qgrid-core/services/dom.js

```diff
@@ -46,4 +47,37 @@ export function eventPath(event) {
 
 export function elementFromPoint(x, y) {
 	return document.elementFromPoint(x, y);
+}
+
+export function selectText(element) {
+	if (!element || !element.classList) {
```

Comment:
it's not a common case to do like that In q-grid, element not null should be tested outside